sqlstore/src/main/resources/db/migration/V8200.sql

```diff
@@ -0,0 +1,22 @@
+-- Create table for instrument ServiceRecords
+CREATE TABLE Instrument_ServiceRecord (
+  recordId bigint(20) NOT NULL,
+  instrumentId bigint(20) NOT NULL,
+  PRIMARY KEY (recordId),
+  PRIMARY KEY (instrumenId),
+  CONSTRAINT fk_instrumentServiceRecord_instrument FOREIGN KEY (instrumentId) REFERENCES Instrument (instrumentId)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8
```

Comment:
* Can only have one primary key (`PRIMARY KEY (recordId, instrumentId)`)
* Typo in `instrumentId` in primary key
* Missing semi-colon at end of statement